halo/app/start_test.go

```diff
@@ -88,15 +88,15 @@ func TestSmoke(t *testing.T) {
 
 	srcChain := netconf.Simnet.Static().OmniExecutionChainID
 	// Ensure all blocks are attested and approved.
-	cprov.Subscribe(ctx, srcChain, 0, "test", func(ctx context.Context, approved xchain.Attestation) error {
+	cprov.Subscribe(ctx, srcChain, 1, "test", func(ctx context.Context, approved xchain.Attestation) error {
```

Comment:
Why do we switch from 0 to 1?

Reply:
We previously attested to all xblocks, including genesis. And we used block height as the cursor (height==0).
Now, we only attest to non-empty blocks. And we use block offset as cursors, and they start at 1 (offset=1).